lib/Common/Memory/HeapBlock.cpp

```diff
@@ -930,6 +930,20 @@ void HeapBlock::PrintVerifyMarkFailure(Recycler* recycler, char* objectAddress,
                     return;
                 }
 
+                // Js::Type::entryPoint may contain outdated data uncleared, and reused by recycler
+                // Most often occurs with script function Type
+                if (offset ==
+#if TARGET_64
+                    0x18
```

Comment:
Thanks for review! Did not use "offsetof" to avoid dependency noise (same reason of a bunch of this pattern in existing code). This is in "memory" lib which so far only includes basic core and memory headers. Will need to introduce runtime headers to use "offsetof" -- will consider. This is for debug and probably temporary as ideally we'd fix such false positives instead of filtering them.